kodi_addon_checker/xml_schema/gotham_metadata.xsd

```diff
@@ -8,20 +8,20 @@
         <xs:element name="description" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="disclaimer" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="platform" type="platformList" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="language" type="nonEmptyString"  minOccurs="0" maxOccurs="1"/>
-        <xs:element name="license" type="nonEmptyString"  minOccurs="0" maxOccurs="1" />
-        <xs:element name="forum" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="website" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="source" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="email" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="broken" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="language" type="nonEmptyStringCapped"  minOccurs="0" maxOccurs="1"/>
+        <xs:element name="license" type="nonEmptyStringCapped"  minOccurs="0" maxOccurs="1" />
+        <xs:element name="forum" type="xs:string" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="website" type="nonEmptyStringCapped" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="source" type="nonEmptyStringCapped" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="email" type="xs:string" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="broken" type="nonEmptyStringCapped" minOccurs="0" maxOccurs="1"/>
```

Comment:
no max length mentioned for any of those elements in the wiki, they can be as long as wanted

Reply:
The initial trigger for this was https://github.com/xbmc/addon-check/pull/118 which was a limit for the news section. We need to change the wiki and at that point we can also think of limiting the other fields. But it's likely that we want shorter limits.

Until then I'm going to not cap them.